packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -213,7 +214,20 @@ module.exports = {
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
-    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+    // Trigger some typechecking if a file matches with an @ flow comment
+    new FlowTypecheckPlugin({
+      flowconfig: [
+        '[ignore]',
+        '<PROJECT_ROOT>/node_modules/fbjs/.*',
+        '',
+        '[libs]',
+        './flow-typed'
```

Comment:
@gaearon By the way, linting in test files will not run as well... I think we need to find a way to run eslint+flow during the test watching... I don't exactly know just yet how to do that but I can always take a look...